### ISSUE TITLE
UTC timezone cannot be changed for category 0 "home"

### ISSUE THREAD

**Issue by psteinb** [indico/indico#1769]
for indico 1.2, the default timezone UTC cannot be changed for the "home" screen (category 0) even though the TZ in general settings is set to a different value, this was reproduced with indico 1.1.2 as well
indico.cern.ch (indico 1.9 at the time of writing) apparently does not show this problem!

**Comment by pferreir** [comment]
I believe that `indico.cern.ch` doesn't show this problem because the TZ was set by hand at some point. By default, the home category will be in UTC. That is normally not an issue since people don't normally add events directly to it.

The timezone for the home category should be configurable, though. We'll have to fix that.

Meanwhile, you can do this (`indico_shell`): 

```python
>>> categ = CategoryManager.getById('0')
>>> categ.setTimezone('Europe/Zurich')  # or whatever timezone you decide
>>> dbi.commit()
```

**Comment by mattboes** [comment]
psteinb and I are colleagues, and we just did this. I can confirm that this worked, although you have to instantiate CategoryManager first.

> >>> cm = CategoryManager()
> >>> categ = cm.getById('0')
> >>> categ.setTimezone('Europe/Zurich')  # or whatever timezone you decide
> >>> dbi.commit()

Now the home category has the correct time zone and seems to propagate it to sub-items correctly. Thanks for the workaround!

**Comment by pferreir** [comment]
Yes, indeed. Sorry, I didn't test the code beforehand :blush:

**Comment by ThiefMaster** [comment]
Fixed in 1.9,x/2.0
